### DISCUSSION TITLE
How do you add custom stones & ores? I've tried adding both a folder and zip file to the globalresources folder in my modpack, but it's not working.

### DISCUSSION THREAD

**Discussion by vizthex123** [lukebemish/excavated_variants#205] [Q&A] [answered]
Is it supposed to be a datapack or something? The readme doesn't really mention it.

I've tried both a datapack and the folder in my modpack's directory, and neither is working.

Here's what I made if ya wanna check it.
[ores.zip](https://github.com/lukebemishprojects/ExcavatedVariants/files/15384481/ores.zip)

**Comment by lukebemish** [comment]
Add to the globalresources directory, yeah. Make sure you're on the latest version for your Minecraft version -- I recently fixed a bug that could prevent this from working. If it still isn't, provide your MC version and a log

**Reply by vizthex123** [reply]
Forge for MC 1.20.1. On the latest release of the mod (v3.0.9)

I've tried both a zip file and regular folder, and neither works. Seems like this version is for NeoForge though? (according to the filename, at least). Might be why it's not working, but it's just a guess at best.

Log:
[latest.log](https://github.com/lukebemishprojects/ExcavatedVariants/files/15384818/latest.log)

**Reply by lukebemish** [reply, marked as answer]
Your zip is the wrong structure... you have it in `data/globalresources/custom/globaldata/high-tech` within that zip. No clue where you're getting that from, it's not the 1.20.1 _or_ the new format. For 1.20.1 (see <https://github.com/lukebemishprojects/ExcavatedVariants/blob/1.20.1/README.md>) you want `excavated_variants/high-tech/variants` and use the format for 1.20.1.

**Reply by vizthex123** [reply]
I got it from that readme, but like I said I *vastly* misinterpreted it lmao.

~~Finally got it working (although it doesn't have textures)~~

_**Edit:** Tweaked the file a bit and now it won't load at all :'c_
[excavated_variants.zip](https://github.com/lukebemishprojects/ExcavatedVariants/files/15386452/excavated_variants.zip)

**Reply by lukebemish** [reply]
Your `xycraft.json5` file isn't valid json5 so...

**Reply by vizthex123** [reply]
of course lol. Never seen a json5 formatted like the defaults are, so i wasn't sure what to make of it.

**Reply by vizthex123** [reply]
finally got it working, but nothing has textures. Do I have to add those manually, or is it bugged?
![image](https://github.com/lukebemishprojects/ExcavatedVariants/assets/31721920/9152bc0d-06f1-4423-b2f9-67fb147d0b5c)

it does work for other ores though, as there's now Kivi variants of pretty much everything.

**Reply by vizthex123** [reply]
finally got it working!!
![image](https://github.com/lukebemishprojects/ExcavatedVariants/assets/31721920/d7f9a6ce-69af-4117-8300-d2474af3aa98)

still doesn't have textures for Xychorium though... i guess it's because they're got lighting effects or something. Do you know if this is fixable or should i just remove Xychorium?
![image](https://github.com/lukebemishprojects/ExcavatedVariants/assets/31721920/5690006d-4c11-449a-9fbf-201b61126a51)

it did work with XyCraft's Aluminum though, so that's nice.
![image](https://github.com/lukebemishprojects/ExcavatedVariants/assets/31721920/22efac41-2760-474e-b308-411293057570)
